build.gradle

```diff
@@ -140,10 +140,16 @@ subprojects {
         //
         // Finds and reports FQNs used in links in the wrong format.
         javadocLinkChecker {
-
             threshold = 0
             reportType = "warn"
         }
+
+        // Right margin line checker.
+        //
+        // Finds and reports about the lines of a code with length above threshold.
+        rightMarginWrappingChecker {
+            threshold = 100
```

Comment:
Why do we still call this parameter `threshold`? Let's make it consistent with the tools internal naming.